resources-generator/build.gradle.kts

```diff
@@ -26,18 +26,19 @@ dependencies {
 }
 
 java {
-    sourceCompatibility = JavaVersion.VERSION_11
-    targetCompatibility = JavaVersion.VERSION_11
```

Comment:
why it's removed? why we should increase our plugin compatibility to 17?

Reply:
AGP 8 does not support Java compatibility below 17.